internal/storage/remote_chunk_manager.go

```diff
@@ -84,11 +98,26 @@ func NewRemoteChunkManager(ctx context.Context, c *config) (*RemoteChunkManager,
 	return mcm, nil
 }
 
+// NewRemoteChunkManagerForTesting is used for testing.
```

Comment:
It would be better to move this to *_test.go or use `mockery` instead.

Reply:
Our unittest project arrangement is not good enough to support cross package testing.
`NewRemoteChunkManagerForTesting` is used by other package unittest.